Core/GDCore/Project/ProjectScopedContainers.h

```diff
@@ -236,6 +230,8 @@ class ProjectScopedContainers {
  private:
   gd::ObjectsContainersList objectsContainersList;
   gd::VariablesContainersList variablesContainersList;
+  gd::VariablesContainer legacyGlobalVariables;
+  gd::VariablesContainer legacySceneVariables;
```

Comment:
These are not references/pointers but values, meaning that ProjectScopedContainers moved from "I'm just a set of lists pointing to things in your project" to "I actually hold stuff, and you will have a bad time if you destroyed me and kept references to things".